execution_engine_testing/test_support/src/lib.rs

```diff
@@ -197,3 +195,45 @@ pub static PRODUCTION_ROUND_SEIGNIORAGE_RATE: Lazy<Ratio<u64>> = Lazy::new(|| {
 });
 /// System address.
 pub static SYSTEM_ADDR: Lazy<AccountHash> = Lazy::new(|| PublicKey::System.to_account_hash());
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use super::*;
+
+    #[test]
+    fn defaults_should_match_production_chainspec_values() {
+        let path = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("resources/chainspec.toml");
```

Comment:
Why not `PRODUCTION_PATH` here? It has the same value

Reply:
Good call.  Done in [f5b6158](https://github.com/casper-network/casper-node/pull/3918/commits/f5b61585e251861774e83409c578ab9f16b9a88a).